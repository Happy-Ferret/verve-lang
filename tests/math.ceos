print(2 + 2)
print(8 - 3)
print(3 * 3)
print(4 / 2)
print(8 % 3)
print(4 < 2)
print(4 > 2)
print(4 <= 4)
print(2 >= 4)
print(4 == 4)
print(2 == 4)
print(3 - 5)
print(-3)
print(!0)
print(!1)
print(1 && 1)
print(1 && 0)
print(0 || 1)
print(0 || 0)
