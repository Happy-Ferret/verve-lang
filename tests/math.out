4
5
9
2
2
0
1
1
0
1
0
-2
-3
1
0
1
0
1
0
